internal/btp/cis/provision.go

```diff
@@ -26,8 +26,36 @@ type ProvisionEnvironment struct {
 }
 
 type KymaParameters struct {
-	Name   string `json:"name"`
-	Region string `json:"region"`
+	Administrators *[]string                 `json:"administrators,omitempty"`
+	AutoScalerMax  *int                      `json:"autoScalerMax,omitempty"`
+	AutoScalerMin  *int                      `json:"autoScalerMin,omitempty"`
+	MachineType    *string                   `json:"machineType,omitempty"`
+	Modules        *KymaParametersModules    `json:"modules,omitempty"`
+	Name           string                    `json:"name"`
+	Networking     *KymaParametersNetworking `json:"networking,omitempty"`
+	Oidc           *KymaParametersOidc       `json:"oidc,omitempty"`
+	Region         string                    `json:"region"`
+}
+
+type KymaParametersModules struct {
+	Default bool `json:"default,omitempty"`
+	List    []struct {
+		Name                 string `json:"name,omitempty"`
```

Comment:
Module Name is mandatory (according to GUI)

Reply:
fixed